x-pack/plugins/cloud_security_posture/common/types.ts

```diff
@@ -47,8 +47,14 @@ export interface ComplianceDashboardData {
   trend: PostureTrend[];
 }
 
+export type Status = 'indexed' | 'indexing' | 'index timeout' | 'not deployed' | 'not installed';
```

Comment:
```suggestion
export type Status = 
| 'indexed'  // a comment when this happens
| 'indexing'  // a comment when this happens
| 'index timeout'  // a comment when this happens
| 'not deployed'  // a comment when this happens
| 'not installed'; // a comment when this happens
```

Reply:
_nit_

`index timeout` -> `index_timeout`
`not installed` - `not_installed`` 
`not deployed` -> `not_deployed`